serialization/Schemas/spdx-v3.json

```diff
@@ -0,0 +1,1061 @@
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": ""
+          }
+        },
+        "completeness": {
+          "$ref": "#/definitions/RelationshipCompleteness",
+          "description": ""
+        }
+      }
+    },
+    "Identity": {
+      "title": "Identity",
+      "type": "object",
+      "description": "Principal identity, unspecified type",
+      "additionalProperties": false,
+      "required": [
+        "identifiedBy"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "identifiedBy": {
+          "type": "array",
+          "description": "",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Identifier",
+            "description": ""
+          }
+        }
+      }
+    },
+    "Person": {
+      "title": "Person",
+      "type": "object",
+      "description": "Extends Identity",
+      "additionalProperties": false,
+      "required": [
+        "identifiedBy"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "identifiedBy": {
+          "type": "array",
+          "description": "",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Identifier",
+            "description": ""
+          }
+        }
+      }
+    },
+    "Organization": {
+      "title": "Organization",
+      "type": "object",
+      "description": "Extends Identity",
+      "additionalProperties": false,
+      "required": [
+        "identifiedBy"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "identifiedBy": {
+          "type": "array",
+          "description": "",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Identifier",
+            "description": ""
+          }
+        }
+      }
+    },
+    "Process": {
+      "title": "Process",
+      "type": "object",
+      "description": "Tool as principal (under discussion)",
+      "additionalProperties": false,
+      "required": [
+        "identifiedBy"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "identifiedBy": {
+          "type": "array",
+          "description": "",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Identifier",
+            "description": ""
+          }
+        }
+      }
+    },
+    "SpdxDocument": {
+      "title": "SpdxDocument",
+      "type": "object",
+      "description": "Statements about a serialized file. \"verifiedUsing\" is required.",
+      "additionalProperties": false,
+      "required": [
+        "element"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "namespace": {
+          "$ref": "#/definitions/IRI",
+          "description": "Default namespace for Element ids in the SPDX file (rdf BASE)"
+        },
+        "prefixes": {
+          "$ref": "#/definitions/NamespaceMap",
+          "description": "Namespace abbreviations used in the SPDX file (rdf PREFIX)"
```

Comment:
In the model, `namespace` is used for an object of type `NamespaceMap`, which here is called `prefixes`. I would prefer to keep the naming consistent with the model.

Reply:
The model does not include the payload namespace.  XML supports both a document default namespace and prefixes to shorten references to other namespaces.  For example, from https://www.w3.org/TR/turtle/#sec-intro:
```
@base <http://example.org/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```
I can change `namespace` to either `base` or `baseNamespace`, but either way it needs to be added to the model.  Do you have a preference?